nav2_dwb_controller/dwb_plugins/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ endif()
 install(TARGETS simple_goal_checker stopped_goal_checker standard_traj_generator
         ARCHIVE DESTINATION lib
         LIBRARY DESTINATION lib
-        RUNTIME DESTINATION lib/${PROJECT_NAME}
+        RUNTIME DESTINATION bin
```

Comment:
libraries should go in bin?

Reply:
Yes, for a shared library (`.DLL` files), in general it should go to the `bin` folder for the install space (which is specified in `RUNTIME DESTINATION`), and the corresponding imported library (`.LIB` files) goes to `lib` folder (which is specified in `ARCHIVE DESTINATION`). On Windows, the symbols information is stored in the imported lib and the MSVC linker is linking against it instead of the shared lib.

REF: https://cmake.org/cmake/help/latest/command/install.html